streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java

```diff
@@ -62,4 +65,23 @@ public void testGetRestoreConsumerConfigs() throws Exception {
         assertEquals(returnedProps.get(ConsumerConfig.CLIENT_ID_CONFIG), "client-restore-consumer");
         assertNull(returnedProps.get(ConsumerConfig.GROUP_ID_CONFIG));
     }
+
+    @Test
+    public void testSerde() {
```

Comment:
Another nitpick: There's no need to prefix test methods with `test*` anymore.  I'd suggest names such as `shouldConfigureSerializerAndDeserializer()`.